### PR TITLE
docs: Add missing parameters to both `useSelectedLayoutSegment` & `useSelectedLayoutSegments`

### PR DESCRIPTION
**Pull request by AhmedBaset**
This PR adds missing parameters to the `useSelectedLayoutSegment` & `useSelectedLayoutSegments` hooks since they take *optional* `parallelRoutesKey`.

### CODE REVIEW

File: docs/02-app/02-api-reference/04-functions/use-selected-layout-segment.mdx [hunk 1]
```diff
@@ -39,10 +39,10 @@ export default function ExampleClientComponent() {
 ## Parameters
 
 ```tsx
-const segment = useSelectedLayoutSegment()
+const segment = useSelectedLayoutSegment(parallelRoutesKey?: string)
 ```
 
-`useSelectedLayoutSegment` does not take any parameters.
+`useSelectedLayoutSegment` accepts *optionally* a [`parallelRoutesKey`](docs/app/building-your-application/routing/parallel-routes#useselectedlayoutsegments), which allows you to read the active segment within that slot.
```
Comment (leerob) [inline docs/02-app/02-api-reference/04-functions/use-selected-layout-segment.mdx:45]:
```suggestion
`useSelectedLayoutSegment` *optionally* accepts a [`parallelRoutesKey`](/docs/app/building-your-application/routing/parallel-routes#useselectedlayoutsegments), which allows you to read the active route segment within that slot.
```

File: docs/02-app/02-api-reference/04-functions/use-selected-layout-segments.mdx [hunk 2]
```diff
@@ -51,10 +51,11 @@ export default function ExampleClientComponent() {
 ## Parameters
 
 ```tsx
-const segments = useSelectedLayoutSegments()
+const segments = useSelectedLayoutSegments(parallelRoutesKey?: string)
 ```
 
-`useSelectedLayoutSegments` does not take any parameters.
+`useSelectedLayoutSegments` accepts *optionally* a [`parallelRoutesKey`](docs/app/building-your-application/routing/parallel-routes#useselectedlayoutsegments), which allows you to read the active segment within that slot.
```
Comment (leerob) [inline docs/02-app/02-api-reference/04-functions/use-selected-layout-segments.mdx:57]:
```suggestion
`useSelectedLayoutSegments` *optionally* accepts a [`parallelRoutesKey`](/docs/app/building-your-application/routing/parallel-routes#useselectedlayoutsegments), which allows you to read the active route segment within that slot.
```

**Review by leerob — DISMISSED** [review]

**Comment by leerob** [comment]
Lint is failing, could you address please?

**Comment by A7med3bdulBaset** [comment]
> Lint is failing, could you address please?

I've formatted with prettier. Can you please approve the changes to let the workflow rerun?

**Review by leerob — APPROVED** [review]

**Comment by A7med3bdulBaset** [comment]
still getting `This workflow is awaiting approval from a maintainer in https://github.com/vercel/next.js/pull/53602`

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 6b3e4262612738d4d6679865eb4f0103b0ea68e7

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 6b3e4262612738d4d6679865eb4f0103b0ea68e7

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Pull request merged**